### PR TITLE
Added an image for Team Blockflojt Heroes

### PR DESCRIPTION
**Pull request by sanlil**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     {
       "name": "Team Blockflöjt Heroes",
       "repo": "https://github.com/diversify/blockflojt",
+      "image": "images/teames/blockflojtheroes.png",
```
Comment (johanbrook) [inline data.json]: `teames -> teams` ? :)

**Comment by johanbrook** [comment]
Thanks a lot! We'd _really_ appreciate a `description` field as well!

**Comment by sanlil** [comment]
Of course, but another guy is in charge of the description :)

**Comment by ljungren** [comment]
Nice, just supplied the description as well

**Comment by johanbrook** [comment]
@Sanlil Seems like there's a merge conflict in `data.json`, since @2L84F8 submitted the `description` fix in another PR. 

Would you mind pulling down `master` in your local repo, merge it into this branch, and push again? Thanks!

**Comment by johanbrook** [comment]
Nvm, I fixed it :)

**Pull request closed without merging**